src/dataProcessor/DataProcessor.java

```diff
@@ -0,0 +1,130 @@
+package dataProcessor;
+
+import graphs.Graph;
+
+public class DataProcessor {       //TODO: when implemented differentiate between directed and undirected,weighted and unweighted edges
+                                    //TODO implement writing output out of given graph
+    public enum Type{
+        adjacencyMatrix, edgeList,incidenceMatrix;  //TODO: implement adjacencyList
+    }
+
+    private Graph inputGraph;
+    private Type inputType;
+
+    //textInput is in form of regular representation of graph such as in algorithm tasks
+
+    public Graph processInput(String textInput,Type tempType) throws IncorrectInputFormatException {    //TODO: is this what we want from this method, just to give Graph out of text and Type?
```

Comment:
Very "generic" name. Consider renaming to something like "makeGraphFromString" / "makeGraphFromFile".